src/nickel.rs

```diff
@@ -19,11 +19,12 @@ use default_error_handler::DefaultErrorHandler;
 /// use nickel::{Nickel, Options};
 /// let mut server = Nickel::new();
 ///
-/// // Don't print to stdout when starting the server.
-/// server.options = Options::default().output_on_listen(false);
+/// // Don't print to stdout when starting the server and force using 8 threads.
+/// server.options = Options::default().output_on_listen(false).thread_count(Some(8));
 /// ```
```

Comment:
I think a rebase would be best in this case as it's a minor change to something that hasn't yet merged. Thanks!